script/prerender/cfn.yml

```diff
@@ -0,0 +1,209 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Description: Rex prerender EC2 spot instance workers
+
+Parameters:
+
+  BucketName:
+    Description: Name of the bucket to store the prerendering output files in
+    Type: String
+    ConstraintDescription: Must be an existing S3 bucket
+
+  BucketRegion:
+    Description: AWS Region of the above bucket
+    Type: String
+    ConstraintDescription: Must be a valid AWS region name
+
+  CodeVersion:
+    Description: Code version to use, e.g. "v4"
+    Type: String
+
+  PublicUrl:
+    Description: Public URL to use, e.g. "/rex/releases/${CodeVersion}/${ReleaseIdSuffix}"
+    Type: String
+    ConstraintDescription: Must be a path starting with /
+
+  ReleaseIdSuffix:
+    Description: Release ID suffix to use, e.g. part of some git hash
+    Type: String
+    ConstraintDescription: Must be the last part of the Release ID, without the CodeVersion
+
+  ValidUntil:
+    Description: Timestamp for when the EC2 Fleet request expires (prerendering timeout)
+    Type: String
+    AllowedPattern: ^\d{4}-\d\d-\d\dT\d\d:\d\d:\d\dZ$
+    ConstraintDescription: Must be a timestamp in the YYYY-MM-DDTHH:MM:SSZ format
+
+Resources:
+
+  Cluster:
+    Type: AWS::ECS::Cluster
+    Properties:
+      ClusterName: !Sub ${AWS::StackName}-cluster
+
+  DeadLetterQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      KmsMasterKeyId: alias/aws/sqs
+      QueueName: !Sub rex-${CodeVersion}-${ReleaseIdSuffix}-prerender-dlq
+      ReceiveMessageWaitTimeSeconds: 20
+      VisibilityTimeout: 0
+      # RedriveAllowPolicy: # leads to circular dependency errors with the worker queue
```

Comment:
This would limit which queues can use this DeadLetterQueue. I don't see a way to set it on creation though, due to the circular dependency issue, only edit it in later.